roofit/roofitcore/inc/RooCompositeDataStore.h

```diff
@@ -37,14 +37,19 @@ class RooCompositeDataStore : public RooAbsDataStore {
   RooCompositeDataStore() ;
 
   // Ctors from DataStore
-  RooCompositeDataStore(RooStringView name, RooStringView title, const RooArgSet& vars, RooCategory& indexCat, std::map<std::string,RooAbsDataStore*> inputData) ;
```

Comment:
Right it's part of the public interface, but it really should be considered private because the data stores are implementation details of RooDataSet/RooDataHist.